### ISSUE TITLE
README out of date

### ISSUE THREAD

**Issue by CWSites** [EndBug/add-and-commit#190]
Readme is out of date, using `v7.0.0` doesn't have `default_author`

Error
```
Unexpected input(s) 'default_author', valid inputs are ['add', 'author_name', 'author_email', 'branch', 'cwd', 'message', 'pull_strategy', 'push', 'remove', 'signoff', 'tag']
```

**Comment by EndBug** [comment]
@all-contributors please add @CWSites for their docs report

**Comment by EndBug** [comment]
ℹ️ Fixed in 44ad49f0a5c579e13e278447239be0566bd5a33e
Thanks for your contribution ❤️
